src/pages/en/reference/configuration-reference.md

```diff
@@ -200,11 +200,19 @@ A markdown page is considered a draft if it includes `draft: true` in its front
   }
 }
 ```
+### host
```

Comment:
See the top of this file: it's autogenerated! Happy to help here since I'm not sure how well documented this system is

Reply:
Ah yes, reading! I usually forget to do that 👍